README.md

```diff
@@ -1,7 +1,7 @@
 Project Ramda
 =============
 
-A practical functional library for Javascript programmers.
+Ramda is the Lego® of functional libraries for Javascript.
```

Comment:
I agree with @buzzdecafe here.

In fact I'm writing a longish article framed around that slogan.  I'd hate to lose it.